components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -193,6 +192,18 @@ - (void)applyStyle:(MDCItemBarStyle *)style {
   }
 }
 
+- (void)updateTitleLines {
+  _titleLabel.numberOfLines = _style.titleNumberOfLines;
+  if (_style.titleNumberOfLines == 1) {
+    _titleLabel.adjustsFontSizeToFitWidth = NO;
+    _titleLabel.minimumScaleFactor = 1;
```

Comment:
This value is ignored if adjustFontSizeToFitWidth.
Consider not changing it here.
If you do want to change, I would recommend going back to the default of 0.0.

Reply:
Done - setting when the UILabel is created.